server/app/routes/categories/index.js

```diff
@@ -0,0 +1,31 @@
+// ******************************************
+// internal API endpoint for categories
+// *******************************************
+
+var router = require('express').Router();
+module.exports = router;
+
+var path = require('path');
+var categoryPath = path.join(__dirname, '../../../db/models/category');
+
+var CategoriesModel = require(categoryPath);
+
+router.get('/', function (req, res) {
+
+	// console.log("SOMETHING");
```

Comment:
NBD, but just a small note for the future, could we clean any test comments? Otherwise it's just a little hard to read. Appreciate it!